test/photometry.jl

```diff
@@ -9,17 +9,20 @@ PARAMS = [
 ]
 
 ELL_APERTURES = [
-    EllipticalAperture
+    EllipticalAperture,
+    EllipticalAnnulus
 ]
 
 ELL_PARAMS = [
-    (3, 3, 0)
+    (3, 3, 0),
+    (3, 3, 0, 5/3)
 ]
```

Comment:
Now that exact method works, just lump into the existing test code (you'll see where it should go).